dev-docs/howto/vpn/templates/statefulset.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "..fullname" . }}-frontend
+  labels:
+    {{- include "..labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "..selectorLabels" . | nindent 6 }}
+      component: frontend
+  template:
+    metadata:
+      labels:
+        {{- include "..selectorLabels" . | nindent 8 }}
+        component: frontend
+    spec:
+      hostNetwork: false
+      initContainers:
+      - name: tproxy-setup
+        image: nixery.dev/busybox/iptables
+        command: ["/bin/sh", "-x", "/entrypoint.sh"] 
+        securityContext:
+          capabilities:
+            add: ["NET_ADMIN"]
+        volumeMounts:
+        - name: tproxy-setup
+          mountPath: "/entrypoint.sh"
+          subPath: "entrypoint.sh"
+          readOnly: true
+      {{- if .Values.wireguard.enabled }}
+      - name: wg-setup
+        image: "nixery.dev/busybox/wireguard-tools"
+        command: ["/bin/sh", "-x", "/etc/wireguard/entrypoint.sh"]
+        securityContext:
+          capabilities:
+            add: ["NET_ADMIN"]
+        volumeMounts:
+        - name: wireguard
+          mountPath: "/etc/wireguard"
+          readOnly: true
+      {{- end }}
+      containers:
+      - name: tproxy
+        image: docker.io/burgerdev/tproxy:latest
```

Comment:
We should put this repo/Dockerfile/Containerfile somewhere publicly. Either your or the companies github. Also, I don't know if there's a difference in the registries (i.e. maybe docker.io has stricter download limitation in comparison to ghcr.io) but I also like that on ghcr.io one can link the container to the repository. This would make the source easily findable.

Reply:
The docker.io image was deployed from a dirty branch. It's productionized at github.com/burgerdev/go-tproxy now, adjusted the image value accordingly.